### PR TITLE
RHEL9 java enablement

### PR DESCRIPTION
**Pull request by avshiliaev**
## What does this PR change?

RHEL9 java enablement

## GUI diff

No difference.

- [x] **DONE**

## Documentation
- No documentation needed: only internal and user invisible changes

- [x] **DONE**

## Test coverage

- [x] **DONE**

## Links

Implements https://github.com/SUSE/spacewalk/issues/18471

- [x] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

**Review by aaannz — APPROVED** [review]
I am not sure if this is all what is needed also adding those thousands of sql data seems strange to me, but I guess this is how it was done.

So LGTM from me.

**Review by mcalmer — CHANGES_REQUESTED** [review]
Schema upgrade is missing

File: schema/spacewalk/upgrade/susemanager-schema-4.4.2-to-susemanager-schema-4.4.3/001-rhnKSInstallType.sql [hunk 1]
```diff
@@ -0,0 +1,6 @@
+insert into rhnKSInstallType (id, label, name) (
+        select sequence_nextval('rhn_ksinstalltype_id_seq'),
+               'rhel_9','Red Hat Enterprise Linux 9'
+        from dual
+        where not exists (select 1 from rhnKSInstallType where  label = 'rhel_9')
+    );
```
Comment (mcalmer) [inline schema/spacewalk/upgrade/susemanager-schema-4.4.2-to-susemanager-schema-4.4.3/001-rhnKSInstallType.sql:6]: Wrong schema migration directory. We are still at susemanager-schema-4.4.1 .
So the migration need to go into a directory with the name 

   susemanager-schema-4.4.1-to-susemanager-schema-4.4.2

I think this is also the reason for the schema migration test errors

File: schema/spacewalk/upgrade/susemanager-schema-4.4.2-to-susemanager-schema-4.4.3/002-rhnKickstartTimezome.sql [hunk 2]
```diff
@@ -0,0 +1,4150 @@
+insert into rhnKickstartTimezone (id, label, name, install_type)
```
Comment (mcalmer) [inline schema/spacewalk/upgrade/susemanager-schema-4.4.2-to-susemanager-schema-4.4.3/002-rhnKickstartTimezome.sql:1]: same as above - wrong schema migration directory

**Review by mcalmer — CHANGES_REQUESTED** [review]

**Review by mcalmer — APPROVED** [review]
Looks good now. You only need to solve the conflict in the changes before you can merge.

**Pull request merged**